connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -89,25 +91,33 @@ components:
                       type: array
                       description: The RECORD/STATE/LOG AirbyteMessages coming from the read operation for this page
                       items:
-                        $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteMessage"
+                        type: object
+                        # $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteMessage"
                     request:
                       $ref: "#/components/schemas/HttpRequest"
                     response:
                       $ref: "#/components/schemas/HttpResponse"
     StreamReadRequestBody:
       type: object
       required:
-        - definition
+        - manifest
         - stream
+        - config
       properties:
-        definition:
-          $ref: "#/components/schemas/ConnectorDefinitionBody"
-          description: The config-based connector definition contents
+        manifest:
+          type: object
+          description: The config-based connector manifest contents
+          # $ref: "#/components/schemas/ConnectorManifest"
         stream:
           type: string
           description: Name of the stream to read
+        config:
+          type: object
+          description: The config blob containing the user inputs for testing
```

Comment:
I realized that this was missing from the `/stream/read` endpoint, so I added it here